btcec/schnorr/musig2/musig2_test.go

```diff
@@ -1017,3 +1143,19 @@ func to32ByteSlice(input []byte) [32]byte {
 	copy(output[:], input)
 	return output
 }
+
+func to66ByteSlice(input []byte) [66]byte {
```

Comment:
Can use `PubNonceSize` here instead of `66`.

Reply:
It's also actually possible to just do a type cast here, but it ends up being pretty ugly, so we usually just make the copy explicit like here.